tsconfig.json

```diff
@@ -21,6 +21,8 @@
     ],
     "allowJs": true,
     "skipLibCheck": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
```

Comment:
these are necessary to get decorators (e.g. `@Column`) to work: https://www.typescriptlang.org/docs/handbook/decorators.html